packages/opentelemetry-instrumentation-llamaindex/tests/test_query_pipeline.py

```diff
@@ -53,7 +52,6 @@ def test_query_pipeline(exporter):
         "LLM.task",
         "OpenAI.task",
         "TokenTextSplitter.task",
-        "cohere.rerank",
```

Comment:
`Cohere.task`?

Reply:
`Cohere.task` is not included because the llm used is OpenAI (so `OpenAI.task`), so only the reranker (`CohereRerank.task`) shows up. It might have been a typo from earlier because the test used to be skipped.